### ISSUE TITLE
Multiple Birdseye instances / views / restreams

### ISSUE THREAD

**Issue by trbmchs** [blakeblackshear/frigate#14371]
**Describe what you are trying to accomplish and why in non technical terms**
I want to be able to combine certan cameras into "birdseye" like views where the camreas are tiled together. 
this would be usefull for checking zones like 
- all basement cameras 
- all first floor cameras
- all exterior camers
this would allow restream of these to specific addresses "rtsp://0.0.0.0:8554/birdseye[1-4]"  
I specifically want the restream of the tiled cameras, not just the camera groups

**Describe the solution you'd like**
extend frigate to have multiple birdseye instances

**Describe alternatives you've considered**
this might be able to be done to the raw feeds using ffmepg hstack/vstack filters, but im not sure.

**Comment by DrSpaldo** [comment]
@trbmchs Great feature request. Would be a good one to have for sure.

**Comment by bagobones** [comment]
Seems like this would be a good option to have as part of the camera groups feature. Basically just add a check box for Birdseye and have it generate one for that group.

**Comment by the-nicolas** [comment]
That looks not to complicated to implement and would be really helpful. I have a small wall screen and just want the external cameras on that and the full birdeye just one the bigger screens.

I think mixing that with the camera groups would make it less flexible. Just like mentioned in the first post a Birdseye number and a per camera defiinion a list of Birdseye view where that camera should get included and the position would be perfect!

**Comment by breakingspell** [comment]
Hello, I opened https://github.com/blakeblackshear/frigate/issues/18088 to address low FPS in Birdseye Restream. It looks like this feature is almost what I would be looking for. 

With this workflow concept (configure camera group and enable birdseye on the group), would it be possible to source the higher-quality `record` streams for each camera?

**Comment by NickM-27** [comment]
No, recording stream won't work. It also generally doesn't make sense that use the recording stream because unless you're outputting an 8K stream, the additional resolution is lost in resizing and decoding anyway.

**Comment by breakingspell** [comment]
Hm, would it be apt to define a low-resolution/high-refresh restream feed just for Birdseye? 

I should be specific that the framerate is the only factor i'd like to increase in Birdseye, not resolution.

**Comment by NickM-27** [comment]
it just doesn't make sense to separately decode the record stream, which is not really feasible for most users hardware to be done across all cameras. Then you have the issue of the streams being out of sync.

If fps is your only concern then you should just increase your detect config to have 10fps, the default is 5 but birdseye supports up to 10fps

**Comment by bagobones** [comment]
> I should be specific that the framerate is the only factor i'd like to increase in Birdseye, not resolution (let's continue discussion on FPS in that issue)

Having frigate render a new RTSP stream at high FPS seems like a lot of server work, if your display device has a browser and you can run home assistant on that browser you might want to consider the new [Grid](https://card.camera/#/screenshots?id=grid) mode on the HACS [Frigate Card / Advanced Camera Card](https://card.camera/) as it will dynamically highlight active streams and provide a compact multi camera view.

**Comment by extremeshok** [comment]
+1 for this

**Comment by brukutu** [comment]
Perhaps keep it simple. 
Keep it exactly like current (detect streams etc) but simply allow user to have a bird eye view stream per camera group? 

I love the idea of birdseye, but when running on a 60+ camera system, with multiple locations, doesn’t workout well. If it could just be “by camera group” it would solve a lot of issues.

**Comment by singhdd93** [comment]
I would love to have this feature.
